sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/issue/NewIssue.java

```diff
@@ -74,6 +81,18 @@ public interface NewIssue {
    */
   NewIssue addFlow(Iterable<NewIssueLocation> flowLocations);
 
+  /**
+   * Register a flow for this issue. A flow is an ordered list of issue locations that help to understand the issue.
+   * It should be a <b>path that backtracks the issue from its primary location to the start of the flow</b>.
+   * Several flows can be registered.
+   *
+   * @param flowType specifies type of flow that is being added. For details see
+   * {@link org.sonar.api.batch.sensor.issue.NewIssue.FlowType FlowType}.
```

Comment:
The class name doesn't need to be fully qualified since it's in the same package.

Reply:
Fixed thanks, actually I didn't know about it, I thought in JavaDoc we always need to reference fully qualified name).